### PR TITLE
Interviewee registration and display added

### PR DESCRIPTION
**Pull request by ManeeshaPerera**
Interviewee registration and displaying interviewed individuals completed

### CODE REVIEW

**Comment by rehrumesh** [comment]
@ManeeshaPerera please resolve the conflicts locally and push the commits. There are conflicts in following files.

web/src/app/pages/pages.module.js
web/src/app/pages/tasks/addtasks.controller.js

File: web/src/app/pages/register/internReg.controller.js [hunk 1]
```diff
@@ -57,12 +57,9 @@
                 },
                 data: data
             }).then(function successCallback(response) {
-<<<<<<< HEAD
-                if (response.data == "SUCCESS") {
-                    console.log(response.data);
-=======
+
                 if (response.data.status == "success") {
```
Comment (rehrumesh) [inline web/src/app/pages/register/internReg.controller.js]: use === instead of ==

File: web/debug.log [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[0914/174641:ERROR:process_info.cc(330)] VirtualQueryEx: Access is denied. (0x5)
```
Comment (rehrumesh) [inline web/debug.log]: IMO log files should be ignored. @AshanFernando  @niroshannrsh

File: web/src/app/pages/register/internReg.controller.js [hunk 3]
```diff
@@ -96,6 +98,35 @@
...
+        $scope.submitted='true'; //for form validation
+        $scope.reset=function(){
+          $scope.submitted=false;
+
+          $scope.formdata={};
+        }
+
+        $scope.University=["UCSC","UOM-CSE","UOM-IT","University of Kelaniya","Uva Wellassa University","University of Rajarata","University of Peradeniya","University of Jaffna","SLIIT","IIT","APIIT","Sabaragamuwa University","Sri Jayawardanapura University","Northshore College of Business and Technology","University of Wayamba","Auston University","General Sir John Kotelawala Defence University","NSBM","Umea University–Sweden"];
+        $scope.addInterviewee= function(){
+          //console.log($scope.formdata);
```
Comment (rehrumesh) [inline web/src/app/pages/register/internReg.controller.js]: Remove commented code.

File: web/src/app/pages/register/internReg.controller.js [hunk 4]
```diff
@@ -96,6 +98,35 @@
...
+          $scope.formdata.id=$scope.formdata.email;
+          var status={'status':'interviewed'};
+          $scope.formdata=angular.merge($scope.formdata,status);
+          console.log($scope.formdata);
```
Comment (rehrumesh) [inline web/src/app/pages/register/internReg.controller.js]: You can remove console.logs from the code.

File: web/src/app/pages/tasks/addtasks.controller.js [hunk 5]
```diff
@@ -7,29 +7,16 @@
...
-            preventDuplicates: false,
-            preventOpenDuplicates: false,
-            title: "",
-            msg: "Task added successfully"
-        };
 
         var getAllTasks = function() {
-            $http.get('http://localhost:3000/dev/tasks/allTask').then(function(response) {
+
+            $http.get('https://ezh9ingj6l.execute-api.us-east-1.amazonaws.com/dev/allTask').then(function(response) {
```
Comment (rehrumesh) [inline web/src/app/pages/tasks/addtasks.controller.js]: API had another URL right? 
Ex: https://github.com/99xt/interns-portal/blob/0c26e458c8d97e4a623c08b4d0afe6b8e765a96f/web/src/app/pages/home/home.controller.js#L38

File: web/src/app/pages/register/internReg.controller.js [hunk 6]
```diff
@@ -81,21 +83,56 @@
...
+        $scope.formdata = {};
+
+        $scope.submitted = 'true'; //for form validation
+        $scope.reset = function() {
+            $scope.submitted = false;
+
+            $scope.formdata = {};
+        }
+
+        $scope.University = ["UCSC", "UOM-CSE", "UOM-IT", "University of Kelaniya", "Uva Wellassa University", "University of Rajarata", "University of Peradeniya", "University of Jaffna", "SLIIT", "IIT", "APIIT", "Sabaragamuwa University", "Sri Jayawardanapura University", "Northshore College of Business and Technology", "University of Wayamba", "Auston University", "General Sir John Kotelawala Defence University", "NSBM", "Umea University–Sweden"];
```
Comment (rehrumesh) [inline web/src/app/pages/register/internReg.controller.js:119]: Store in the DB?

Comment (ManeeshaPerera) [reply web/src/app/pages/register/internReg.controller.js:119]: Anura aiya told to store the data in the front end at the moment. Shall I change?

**Review by rehrumesh — APPROVED** [review]

**Pull request merged**